.github/workflows/sub-build-docker-image.yml

```
@@ -21,19 +21,17 @@ on:
       rust_lib_backtrace:
         required: false
         type: string
+      # defaults to: vars.RUST_LOG
       rust_log:
         required: false
         type: string
-        default: info
-      # keep these in sync with:
-      # https://github.com/ZcashFoundation/zebra/blob/main/docker/Dockerfile#L83
+      # defaults to: vars.RUST_PROD_FEATURES
       features:
         required: false
-        default: "default-release-binaries"
         type: string
+      # defaults to: vars.RUST_TEST_FEATURES and vars.RUST_PROD_FEATURES
```

Looking at line 53, this seems to default only to `vars.RUST_TEST_FEATURES`. I made a suggestion.

---

This default happens inside `entrypoint.sh`, not the workflow.

```suggestion
      # defaults to: vars.RUST_TEST_FEATURES (and entrypoint.sh adds vars.RUST_PROD_FEATURES)
```